mobile-sdk/android/ui-customization/java/ui-customization.java

```diff
@@ -0,0 +1,58 @@
+  // call this before u display the ui sdk on the screen
+  private void configureDojoThemeSettings(){
+    // you can override any value you for lightColorPalette
+    LightColorPalette lightColorPalette = new  LightColorPalette(
+            "#DD000000",
+            "#DD000000",
+            "#DD000000",
+            "#DD000000",
+            "#DD000000",
+            "#DD000000",
+            "#DD000000",
+            "#DD000000",
+            "#DD000000",
+            "#DD000000",
+            "#DD000000",
+            "#DD000000",
+            "#DD000000",
+            "#DD000000",
+            "#DD000000",
+            "#DD000000",
+            "#DD000000",
+            "#DD000000",
+            "#DD000000",
+            "#DD000000");
+
+    // you can override any value for the darkColorPalette
+    DarkColorPalette darkColorPalette = new DarkColorPalette(
+            "#DD000000",
```

Comment:
ask @Saamyyy to give you the updated version of this with // comments for each property